guildwars2/notifiers.py

```diff
@@ -1021,3 +1038,12 @@ async def forced_account_names(self):
     @forced_account_names.before_loop
     async def before_forced_account_names(self):
         await self.bot.wait_until_ready()
+
+    async def db_find_forum_post(self, title):
+        return await self.bot.database.db.gw2.updates.find_one({"title": title})
+
+    async def db_insert_forum_post(self, title):
+        await self.bot.database.db.gw2.updates.insert_one({"title": title, "count": 0})
+
+    async def db_update_forum_post(self, title, count):
+        await self.bot.database.db.gw2.updates.update_one({"title": title}, { "$set": {"title": title, "count": count} })
```

Comment:
These functions likely won't be reused, so I think they'd be better off as inner functions for the caller, or simply not existing

Reply:
Made them an inner function